sdk/storage/storage-blob/samples/javascript/advanced.js

```diff
@@ -61,11 +57,12 @@ async function main() {
   // Parallel uploading a Readable stream with uploadStreamToBlockBlob in Node.js runtime
   // uploadStreamToBlockBlob is only available in Node.js
   await uploadStreamToBlockBlob(
-    Aborter.timeout(30 * 60 * 60 * 1000), // Abort uploading with timeout in 30mins
+    Aborter.timeout(30 * 60 * 1000), // Abort uploading with timeout in 30mins
```

Comment:
Make similar changes in the corresponding typescript files

Reply:
https://github.com/Azure/azure-sdk-for-js/pull/2317/commits/fb59ae0d107a374c220b04f78331e555de8d695a